### ISSUE TITLE
[stable/mariadb] clusterIp provided IP is already allocated

### ISSUE THREAD

**Issue by cronolio** [helm/charts#15922]
[master-svc.yaml](https://github.com/helm/charts/blob/master/stable/mariadb/templates/master-svc.yaml#L19) and [slave-svc.yaml](https://github.com/helm/charts/blob/master/stable/mariadb/templates/slave-svc.yaml#L20) is use the same clusterIP from the [values-production.yaml](https://github.com/helm/charts/blob/master/stable/mariadb/values-production.yaml#L68) (or [values.yaml](https://github.com/helm/charts/blob/master/stable/mariadb/values.yaml#L68)) So finally it say
```
Error: release maria failed: Service "maria-mariadb-slave" is invalid: spec.clusterIP: Invalid value: "xx.xx.xx.xx": provided IP is already allocated

**Comment by Aisuko** [comment]
cc @carrodher Hello sir, Is this has some special settings for production deployments?

**Comment by cronolio** [comment]
Nope. Difference is described [here](https://github.com/helm/charts/tree/master/stable/mariadb#production-configuration)

**Comment by Aisuko** [comment]
Thanks.

**Comment by miguelaeh** [comment]
Hi @cronolio ,
I think that PR will fix the problem you are facing. Could you confirm please?

**Comment by cronolio** [comment]
hm. possible problem is that production values has ```slave.replicas: 2```
